008-infinispan-getting-started/src/main/resources/META-INF/hotrod-client.properties

```diff
@@ -0,0 +1,9 @@
+# Configuration file
+quarkus.http.port=8081
+infinispan-client.hotrod.server-list=localhost:11222
+# Authhotrod.
+infinispan.client.hotrod.auth_server_name=infinispan
+infinispan.client.hotrod.auth_realm=default
+infinispan.client.hotrod.auth_username=my_username
+infinispan.client.hotrod.auth_password=my_password
+infinispan.client.hotrod.sasl_mechanism=DIGEST-MD5
```

Comment:
Add this to make this work on macOS / Windows
```
## Docker Desktop requires this because network is not shared between host and the VM
## error that client gives: ISPN004015: Failed adding new server 172.17.0.2:11222
infinispan.client.hotrod.client_intelligence=BASIC
```